lib/generateNormals.js

```diff
@@ -128,7 +129,11 @@ function generateFaceNormals(gltf, primitive, positionSemantic) {
                     Cartesian3.subtract(positionThree, positionOne, scratchNormalTwo);
                     var normal = new Cartesian3();
                     Cartesian3.cross(scratchNormalOne, scratchNormalTwo, normal);
-                    Cartesian3.normalize(normal, normal);
+                    if (Cartesian3.equalsEpsilon(normal, Cartesian3.ZERO, CesiumMath.EPSILON7)) {
+                        Cartesian3.clone(Cartesian3.UNIT_X, normal);
+                    } else {
+                        Cartesian3.normalize(normal, normal);
+                    }
```

Comment:
In https://github.com/AnalyticalGraphicsInc/cesium/pull/5010, we decided to go for a stage for degenerating these triangles. I think we should make an issue for that and not use this code for consistency.